src/openrct2/interface/Widget.h

```diff
@@ -60,6 +61,62 @@ enum
 
 constexpr uint8_t SCROLLBAR_WIDTH = 10;
 
+constexpr rct_widget MakeWidget(
+    ScreenCoordsXY origin, ScreenSize size, uint8_t type, uint8_t colour, uint32_t content = 0xFFFFFFFF,
+    rct_string_id tooltip = STR_NONE)
+{
+    rct_widget out = {};
+    out.left = origin.x;
+    out.right = origin.x + size.width - 1;
+    out.top = origin.y;
+    out.bottom = origin.y + size.height - 1;
```

Comment:
I don't quite get why providing the `width` and the `height` with extra 1s and then subtracting immediately

Reply:
Future proofing, really. Right now, they're not necessary, as we translate them back to the old struct — for now. However, if left = right, you'd still want width = 1. Likewise, if top = bottom, you'd want height = 1.